api/HTMLInputElement.json

```diff
@@ -693,7 +693,7 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "5.1"
+              "version_added": "5"
             },
             "safari_ios": {
               "version_added": "5"
```

Comment:
Does this also need to change? Per https://github.com/mdn/browser-compat-data/pull/11299 the pairing should be 5/4.2, not 5/5.